guide/spanish/linux/10-simple-and-useful-linux-commands/index.md

```diff
@@ -6,16 +6,17 @@ localeTitle: 10 comandos de Linux simples y útiles
 
 Los comandos que se enumeran aquí son básicos y lo ayudarán a comenzar rápidamente. Pero también son poderosos, y seguirán siendo útiles a medida que se expanda su experiencia en Linux.
 
-1.  `echo` Esto toma el texto que le das y lo envía a algún lugar: para volver a la pantalla, a un archivo u otro comando. Ejemplo: `echo "hello!"`
+1.  `echo` Esto toma el texto que le das y lo envía a algún lugar: de vuelta a la pantalla, a un archivo u otro comando. Ejemplo: `echo "hello!"`
```

Comment:
Esto estara tomando tu texto y lo estara enviando a un lugar adecuado: Para poder regresar a la pantalla principal, a un archivo u otro comando. 

This is the right translation.